libs/application-generic/src/usecases/merge-preferences/merge-preferences.usecase.ts

```diff
@@ -1,9 +1,8 @@
+import { merge } from 'lodash';
```

Comment:
For 100% manual tree shaking

```suggestion
import merge from 'lodash/merge';
```

Reply:
ahh yeah missed that